subnet/subnet_test.go

```diff
@@ -65,3 +68,68 @@ func TestSubnetDeploy(_ *testing.T) {
 	blockchainID, _ := newSubnet.Commit(*deployChainTx, wallet, true)
 	fmt.Printf("blockchainID %s \n", blockchainID.String())
 }
+
+func TestSubnetDeployMultiSig(_ *testing.T) {
+	subnetParams := getDefaultSubnetEVMGenesis()
+	newSubnet, _ := New(&subnetParams)
+	network := avalanche.FujiNetwork()
+
+	keychainA, _ := keychain.NewKeychain(network, "KEY_PATH_A")
```

Comment:
can we use env vars here? for KEY_PATH_A, so the example can be executed without changes

Reply:
ok I saw it is not needed.. anyway maybe it can check for a path or an env var containing a path... I am not sure if that results in less or more confusion